### ISSUE TITLE
export of properties does not work

### ISSUE THREAD

**Issue by jmesserly** [dart-lang/dev_compiler#549]
the fix for https://github.com/dart-lang/dev_compiler/issues/543 did not catch this new example:
foo.dart:
```dart
import 'bar.dart' as bar;

final y = 38;
int get info => 42;

void main() {
  print(bar.info + y);
}
```
bar.dart:
`export 'foo.dart';`

we don't know when we have a synthetic field that we need to ask loader to emit the getter and the setter individually.
